ninja-core/src/test/java/ninja/utils/CookieDataCodecTest.java

```diff
@@ -184,4 +191,32 @@ public void decode_values_with_gibberish_in_them() throws UnsupportedEncodingExc
         decode(outMap, data);
         assertEquals(0, outMap.size());
     }
+
+    @Rule public ExpectedException thrown = ExpectedException.none();
+    /* testedClasses: CookieDataCodec */
+    /*
+     * Test generated by Diffblue Deeptest.
+     * This test case covers the entire method.
+     */
+    @Test
+    public void constructorOutputVoid() {
```

Comment:
discard

Reply:
Discard. Method under test is a constructor